integrations/ultralytics/train.py

```diff
@@ -535,6 +539,11 @@ def train(hyp, opt, device, tb_writer=None, wandb=None):
         help="set flag to enable Automatic Mixed Precision (AMP). disabled by default "
         "in SparseML integration"
     )
+    parser.add_argument(
+        "--leaky-relu",
+        action="store_true",
```

Comment:
given that our default is LeakyRelu for now, would it make sense to change this to store_false along with a negative on the name?

Reply:
Looks good to me from a usability perspective.  I changed it to something a little more verbose than `no-leaky-relu` because there are still configs that can generate a model that uses leakyrelu that this arg wouldn't affect.